Accept Fractal Paginators and Custom Resource Names

eblount opened the pull request.

freekmurze commented:
On first glance, this look like a quality PR :+1: I'll review it thoroughly within a couple of days. Could you rebase it against the current master?

eblount commented:
Thanks! I've rebased against your master. The only conflict was the line I corrected in `__call` in `Fractal.php`, where it looks like an older variable name was used in the `trigger_error` log message.

freekmurze commented:
Very nice! Thanks for your contribution!

eblount commented:
Thanks for the quick acceptance and Twitter shoutout!
